### PR TITLE
Update templates for primary and segments HDUs

### PR DESCRIPTION
**Pull request by menanteau**
Remove some -99 values in template
Remove some '' values in template
Clean up template to represent as undefined all KEYWORDS that are expected as meta-data from SAL Events/Telemetry.

### CODE REVIEW

File: etc/ATSCam/primary_hdu.header [hunk 1]
```diff
@@ -3,56 +3,56 @@ BITPIX  =                    8 / bits per data value
...
-DATE-OBS= ''                   / Date of the observation (image acquisition)
-MJD     =                -99.0 / Modified Julian Date that the file was written
-MJD-OBS =                -99.0 / Modified Julian Date of observation
+DATE    =                      / Creation Date and Time of File
+DATE-OBS=                      / Date of the observation (image acquisition)
+MJD     =                      / Modified Julian Date that the file was written
+MJD-OBS =                      / Modified Julian Date of observation
 OBS-LONG=           -70.749417 / [deg] Observatory east longitude
 OBS-LAT =           -30.244639 / [deg] Observatory latitude
 OBS-ELEV=               2663.0 / [m] Observatory elevation
```
Comment (timj) [inline etc/ATSCam/primary_hdu.header:12]: General comment, but we have to add in OBSGEO headers reasonably soon.

File: etc/ATSCam/primary_hdu.header [hunk 2]
```diff
@@ -3,56 +3,56 @@ BITPIX  =                    8 / bits per data value
...
-OBS-NITE= ''                   / Night of the observation
+OBS-NITE=                      / Night of the observation
 TELESCOP= 'LSST AuxTelescope'  / Telescope name
 INSTRUME= 'ATSCAM'             / Instrument used to obtain these data
```
Comment (timj) [inline etc/ATSCam/primary_hdu.header]: Please update to LATISS.

**Review by timj — APPROVED** [review]
These changes to the templates are fine with me, although it made me question some other things about the headers.

Please fix the bad merge commit before merging to master: "Merge branch 'tickets/DM-15001' of https://github.com/lsst-dm/HeaderS...".  `git rebase master` should do it for you.

File: etc/ATSCam/primary_hdu.header (hunk 1 above)
Comment (menanteau) [reply etc/ATSCam/primary_hdu.header:12]: Will add this to audit ticket

File: etc/ATSCam/primary_hdu.header (hunk 2 above)
Comment (menanteau) [reply etc/ATSCam/primary_hdu.header]: Updated

**Pull request merged**